python/ray/data/_internal/stats.py

```diff
@@ -1173,11 +1173,13 @@ def to_string(self) -> str:
         ):
             out += "\nDataset iterator time breakdown:\n"
             if self.block_time.get():
-                out += "* Total time user code is blocked: {}\n".format(
-                    fmt(self.block_time.get())
+                out += (
+                    "* Total time user thread is blocked by iter_batches: {}\n".format(
+                        fmt(self.block_time.get())
+                    )
                 )
             if self.user_time.get():
-                out += "* Total time in user code: {}\n".format(
+                out += "* Total execution time for user thread: {}\n".format(
```

Comment:
This is a personal opinion, but i feel that "user code" is more intuitive than "user thread" for the purpose of debugging logs. We should see what others think about the exact text we use

Reply:
@stephanie-wang @c21 thoughts on the text description? "user code" vs "user thread" or any other alternative?